src/slic3r/GUI/WebGuideDialog.cpp

```diff
@@ -1139,8 +1139,8 @@ int GuideFrame::LoadProfile()
                 //cout << iter->path().string() << endl;
 
                 wxString strVendor = from_u8(iter->path().string()).BeforeLast('.');
-                strVendor          = strVendor.AfterLast( '\\');
-                strVendor          = strVendor.AfterLast('\/');
+                strVendor          = strVendor.AfterLast('\\');
+                strVendor          = strVendor.AfterLast('/');
```

Comment:
Is it suppose to be escaped?

Reply:
I think regular slash `/` doesn't need ecsaping

Reply:
> regular slash `/` doesn't need ecsaping

Exactly.

Reply:
cool
thanks